README.md

```diff
@@ -30,16 +30,18 @@ You'll also need to clone a new `.env` file from the `.env.tempalate` to store l
 $ cp .env.template .env  # (first time only)
```

Comment:
You've deleted the `.env.template` file, but you still use it for setup so you probably want to re-add that file

Reply:
will add that back in